spec/vehicle_spec.rb

```diff
@@ -1,36 +1,67 @@
 require 'spec_helper'
 
-RSpec.describe Vehicle do
+RSpec.describe Facility do
```

Comment:
Hmm... I'm confused because this file is the vehicle_spec.rb file, where are the vehicle tests?